gamemodes/sffa.pwn

```diff
@@ -46,10 +41,19 @@ native IsValidVehicle(vehicleid);
 #include <SIF/extensions/craft>
 #include <SIF/extensions/debug-labels>
 #include <SIF/old/Dispenser.pwn>
+
+#include <mathutil>
 #include <Balloon>					// By Southclaw:			https://gist.github.com/Southclaw/6254507
-#include <Line>						// By Southclaw:			https://gist.github.com/Southclaw/6254512
+#include <Linegen>					// By Southclaw:			https://gist.github.com/Southclaw/6254512
 #include <Zipline>					// By Southclaw:			https://gist.github.com/Southclaw/6254523
```

Comment:
Most of these includes changed case too, `Line` > `linegen` so this will cause issues on Unix.

Reply:
Yeah but if you built this on Linux or Mac, it would fail because the actual filename is `linegen.inc` not `Linegen.inc`: https://github.com/Southclaws/samp-linegen/blob/master/linegen.inc